Unit test failure - test_post_submit

The following test_post_submit unit test fails on the master branch:

```
======================================================================
FAIL: test_post_submit (app.tests.SubmitNewLicenseViewsTestCase)
POST Request for submit a new license
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/spdx-online-tools/spdx-online-tools/src/app/tests.py", line 1229, in test_post_submit
    self.assertEqual(resp.status_code,200)
AssertionError: 500 != 200
```

--- a/src/app/tests.py
+++ b/src/app/tests.py
@@ -1337,7 +1337,7 @@ class SubmitNewLicenseViewsTestCase(TestCase):
         self.exampleUrl = "testUrl"
         self.sourceUrl = "http://landley.net/toybox/license.html"
         self.urls = [self.sourceUrl]
-        self.osiApproved = "no"
+        self.osiApproved = "Unknown"
         self.comments = "Test Comment"
         self.notes = ""
         self.licenseHeader = ""
@@ -1578,7 +1578,7 @@ class SubmitNewLicenseNamespaceViewsTestCase(TestCase):
         self.sourceUrl = "http://landley.net/toybox/license.html"
         self.license_list_url = self.sourceUrl
         self.github_repo_url = self.sourceUrl
-        self.osiApproved = "no"
+        self.osiApproved = "Unknown"
         self.comments = ""
         self.notes = ""
         self.licenseHeader = ""

